LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019 dc-square GmbH
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
```suggestion
   Copyright 2022-now HiveMQ
```

Reply:
The LICENSE file should not be changed, it only contains the template for the header.
The actual header is in the HEADER file.